docs/content/guides/rows/row-virtualization.md

```diff
@@ -23,18 +23,10 @@ Virtualization allows Handsontable to process hundreds of thousands of records w
 
 This feature is enabled by default and can be turned off by setting the [`renderAllRows`](@/api/options.md#renderallrows) option to `true`.
 
-::: tip
-Note that the data grid without virtualization enabled will only work with relatively small data sets.
-:::
-
 ## Configuring row virtualization
 
 You can experiment with the [`viewportRowsRenderingOffset`](@/api/options.md#viewportrowsrenderingoffset) configuration option, which determines the number of rows displayed outside the visible viewport. If the number passed to that option is greater than the total columns in your data set, then the virtualization will be practically turned off.
 
-::: warning
-Proceed with caution, as it will affect the overall performance of the grid.
-:::
-
```

Comment:
Same question as with `column-virtualization.md`.

Reply:
Same as above, I would keep it that way.